### ISSUE TITLE
I know JS but cant build my own library or tool

### ISSUE THREAD

**Issue by fiftify** [kentcdodds/ama#449]
I have access to your courses on frontendmasters.com and spend lot of time understanding javascript but when it comes to building some tools like many people do i can't.

How do you break down a problem and think about the solution?

Thanks...

**Comment by kentcdodds** [comment]
Hi! I'm afraid I don't have time to give a super solid answer to all my open questions right now, so I just went through and answered all my AMA questions in the form of a video: https://www.youtube.com/watch?v=4Yq6PO0r-fU&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u

A few things that folks can do to help make things better for others reading the questions and answers:

1. Link to when I start answering the question. Add `&t=xxmxxs` to the end of the URL like this: https://www.youtube.com/watch?v=4Yq6PO0r-fU&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u&t=2m5s
2. Comment with links I referenced
3. Summarize what I talk about

Thanks!
